package.json

```diff
@@ -138,7 +140,10 @@
       "graphql": "^16.6.0"
     }
   },
-  "engines": {
-    "node": ">=20.x"
+  "lint-staged": {
+    "**/*.{ts,json}": [
+      "prettier --write"
+    ],
+    "**/*.ts": "eslint --fix"
```

Comment:
Aren't there other extensions that we need to run this script on? Also would these respect the `eslint.ignore` and `prettier.ignore` files?

Reply:
1.According to the current scripts namely "format:fix" , it was run for ts,tsx,json.scss,css but as the backend contains only ts and json and no css and tsx files, I have omitted it from lint-staged. 
2. Yes it does respect the .eslintignore and .prettierignore files.